components/gfx/text/shaping/harfbuzz.rs

```diff
@@ -418,7 +418,7 @@ unsafe impl Sync for FontFuncs {}
 lazy_static! {
     static ref HB_FONT_FUNCS: FontFuncs = unsafe {
         let hb_funcs = hb_font_funcs_create();
-        hb_font_funcs_set_glyph_func(hb_funcs, Some(glyph_func), ptr::null_mut(), None);
+        hb_font_funcs_set_nominal_glyph_func(hb_funcs, Some(glyph_func), ptr::null_mut(), None);
```

Comment:
If you implement `nominal_glyph_func`, then you should also implement `variation_glyph_func`, otherwise you lose support for Unicode Variation Selectors.

Reply:
@khaledhosny Thanks for the info!

I know very little about harfbuzz - the old code that I am porting from ignored the variation parameter in the deprecated glyph callback function.

Does that mean that we weren't previously handling this case anyway? If that's the case we could land this first and do the variation changes as a separate piece of functionality if we're not already supporting it.

Or will this change mean we're losing some *existing* functionality?

Reply:
Sounds about right, old code didn’t handle it indeed.